pkg/api/validation/validation.go

```diff
@@ -1463,8 +1463,6 @@ func validateVolumeMounts(mounts []api.VolumeMount, volumes sets.String, fldPath
 		}
 		if len(mnt.MountPath) == 0 {
 			allErrs = append(allErrs, field.Required(idxPath.Child("mountPath"), ""))
-		} else if strings.Contains(mnt.MountPath, ":") {
```

Comment:
Actually, it seems that:
* `c:/my/path` works with Windows Containers (there's some translation in place)
* `/my:path` is valid Unix path

Please, advise.